clang/include/clang/Basic/AttrDocs.td

```diff
@@ -5796,12 +5796,18 @@ declared entity. The entity must not have weak linkage; for example, in C++,
 it cannot be applied to a declaration if a definition at that location would be
 considered inline.
 
-Not all targets support this attribute. ELF target support depends on both the
-linker and runtime linker, and is available in at least lld 4.0 and later,
-binutils 2.20.1 and later, glibc v2.11.1 and later, and FreeBSD 9.1 and later.
-Mach-O targets support it, but with slightly different semantics: the resolver
-is run at first call, instead of at load time by the runtime linker. Targets
-other than ELF and Mach-O currently do not support this attribute.
+Not all targets support this attribute:
+
+- ELF target support depends on both the linker and runtime linker, and is
```

Comment:
Pending the outcome of the other thread, you should mention baremetal avr here.

Reply:
added, due to it works in the last few releases so deserves an entry.
https://godbolt.org/z/K5dY7qbjj

Reply:
I think we can drop the mention of lld here. lld 4.0 is not quite usable for a lot of issues/missing features.